.github/container/test-maxtext.sh

```diff
@@ -191,7 +224,7 @@ export BASE_XLA_FLAGS=${BASE_XLA_FLAGS:---xla_gpu_enable_latency_hiding_schedule
                 --xla_gpu_enable_highest_priority_async_stream=true
                 --xla_gpu_all_reduce_combine_threshold_bytes=1073741824 
                 --xla_gpu_all_gather_combine_threshold_bytes=1073741824 
-                --xla_gpu_reduce_scatter_combine_threshold_bytes=1073741824
+                --xla_gpu_reduce_scatter_combine_threshold_bytes=134217728
```

Comment:
Why change this in this PR?

Reply:
For RS, the combiner threshold represents the output buffer size. Typically, the `msg size=output_buffer_size*num_gpus`. Hence, to get an effective RS msg size of 1G, we can nominally set it to 128MB assuming 8 gpus on 1 node. For multinode runs, all these values need to be tuned accordingly for best perf.

Reply:
Ok. So this values is a better default value. But it still need to be tuned.